README.md

```diff
@@ -71,11 +71,11 @@ Details:
 - Written in [Java](https://java.com/),
```

Comment:
Keep run configs XML files above too, change the command to use gradle instead.
Also do the same for the other run configs by intellij below.
(Somehow couldn't add a file comment thus added it here)

Reply:
@Osiris-Team, do you have a guide for that? And you mean the .run directory, right? That XML forwards to a lot of Maven-specific blocks.

Reply:
Okay thanks. I'll try it out tomorrow in IntelIJ IDEA CE on my Xubuntu environment and see whether I can get the run tasks back but for Gradle.